### ISSUE TITLE
Regarding SSM block

### ISSUE THREAD

**Issue by sydat2701** [MzeroMiko/VMamba#332]
Hi! Thank you for your great work!
I have a question that hope you can explain. As your paper shows that the SS2D includes 3 phases: cross-scan, S6 blocks, and cross-merge. However, I can not find the S6 block explicitly in SS2Dv2 code. So is the SSM implemented inside the selective_scan function? (which I cannot access to).

**Comment by MzeroMiko** [comment]
Yes. The S6 block is mainly adopted from tridao's mamba, so does the cuda implementation. And we have only some slight modifications on it. You can find the detailed implementation in kernels/selective_scan.
